shithead_game_server/src/lobby.rs

```diff
@@ -73,37 +79,63 @@ impl LobbyId {
 #[derive(Debug, Serialize, PartialEq, Eq, Clone, Copy, Hash)]
 pub enum LobbyState {
     Waiting,
-    Started,
+    GameStarted,
+}
+
+/// A deck of cards
+#[derive(Debug)]
+pub struct Deck {
+    cards: Vec<CardId>,
+}
+impl Deck {
+    /// Creates a shuffled deck of cards.
+    pub fn shuffled() -> Self {
+        // first collect into a vector so that we can shuffle it.
+        let mut card_ids = ALL_CARD_IDS.clone();
+        card_ids.shuffle(&mut rand::thread_rng());
+
+        // after we have a shuffled vector of card ids, we can collect it into a DashSet to
```

Comment:
Where's the DashSet though?

Reply:
No need the dashset since a lobby is already in the dashmap of lobbies, so it is protected from multi threaded accesses.